test/test_load.py

```diff
@@ -11,7 +11,7 @@
 from pyscitt.client import Client
 
 NUM_CLAIMS = 100
-LOCUST_PEAK_USERS = 500
+LOCUST_PEAK_USERS = 100
 LOCUST_USERS_SPAWN_RATE = 10
```

Comment:
nit: since we have decreased the peak number of users, it might make sense to decrease a little the spawn rate (like 5).

Reply:
Thanks, decreased to 5